validator/src/main/resources/expected-data-template/java/eks/rds-mysql-trace.mustache

```diff
@@ -50,6 +50,32 @@
                     "aws.local.operation": "^GET /mysql$"
                 },
                 "subsegments": [
+                    {
+                        "name": "^mysql$",
+                        "sql": {
+                            "url": "^Create connection$",
+                            "database_type": "^mysql$"
+                        },
+                        "annotations": {
+                            "aws.remote.operation": "^UnknownRemoteOperation$",
+                            "aws.local.operation": "^GET /mysql$",
+                            "aws.remote.resource.type": "^DB::Connection$",
+                            "aws.remote.resource.identifier": "^{{remoteResourceIdentifier}}$",
+                            "aws.remote.service": "^mysql$",
+                            "aws.local.service": "^{{serviceName}}$",
+                            "aws.local.environment": "^eks:{{platformInfo}}/{{appNamespace}}$"
```

Comment:
We don't care much about these things, as long as we know the first span is for Create connection.